tests/test_import.py

```diff
@@ -156,3 +156,23 @@ def test(self):
             self.assertEqual(0, ret)
 
         self.assertEqual(1, remove_outs_call_counter.mock.call_count)
+
+
+class TestImportFilename(TestDvc):
+    def setUp(self):
+        super(TestImportFilename, self).setUp()
+        tmp_dir = self.mkdtemp()
+        self.external_source = os.path.join(tmp_dir, "file")
+        with open(self.external_source, "w") as fobj:
+            fobj.write("content")
+
+    def test(self):
+        ret = main(["import", "-f", "bar.dvc", self.external_source])
```

Comment:
We need to test that it support subdirectories properly.

Reply:
Do we need to create the subdirectories when they are not exist?

Reply:
good question! I think it should behave the same way as `dvc add` and `dvc run`